data/ship builders jobs.txt

```diff
@@ -0,0 +1,132 @@
+			label "series name"
+			`	She continues to talk about the ship, "The new series will be called Hauler VI and it will look like two Hauler III series side-by-side, but the cockpit and engine construction will obviously be a lot more complex."`
+			label "cost effective"
+			`	"The Hauler VI will have twice the cargo capacity of a III series." She thinks for a moment, "In it's current design the VI would however require more than twice the crew and we obviously needs to fix that before we can put it into the market."`
+			choice
+				`	"Ok, and how will it perform in combat?"`
+					goto "combat effective"
+				`	"Ok, and what help do you need from me?"`
+					goto "help request"
+			label "combat effective"
+			`	"The VI will have twice the hull of a III-series, but it is also 25% slower and we are still working to get the shields up to the same level as the III-series."`
+			label "help request"
+			`	Ms. Oxygenpocket pauses for a moment and then continues; "We have an Hauler III that we upgraded with the cargo-handling systems for the VI so that we can test those new systems.`
+			`Normally our own department would escort such prototypes, but there is a mole in our organisation and we have very strong competition from the Syndicated Shipyards.`
+			`I only need you to help us travel to the nearby planet Skymoot to deliver sushi for a formal dinner. You will be paid for your expenses.`
+			`Will you help us?"`
+			choice
+				`	"Isn't it dangerous to transport food in an untested prototype?`
+					goto "risky transport"
+				`	"Sure, I'll help you."`
+					accept
+				`	"Sorry, I don't have time to help you."`
+					decline
+			label "risky transport"
+			`	"No it's actually more of a formality. Those systems have gone through a lot of simulation cycles already."`
+			choice
+				`	"I'll help you."`
+					accept
+				`	"Sorry, I don't have time to help you."`
+					decline
+	on complete
+		dialog
+			`You are waiting for the Hauler prototype to land, but it appears to stay in orbit.`
+			`There is no response to your hails by radio. Looks like you need to take off and investigate.`
+
+
+mission "Escort Hauler prototype to student party"
+	description "Escort the ship with the new cargo-systems to <destination> to allow it to deliver cooked fish dishes to an archeology students party."
+	source "Skymoot"
+	destination "Zug"
+	landing
+	to offer
+		has "Escort Hauler prototype to formal dinner: offered"	
+	npc accompany
+		government "Merchant"
+		ship "Hauler III" "SB-Labs Cargo Cooker"
+	on enter
+		dialog
+			`As soon as you take off you receive an angry voice message over a short-range inter-ship communications channel: "Why were you trying to contact me over radio! Does everybody here need to know that we are flying around here???`
+			`Let's go to <destination>." You wonder why the destination changed.`
```

Comment:
This should have a quotation mark at the beginning, because it's part of character dialogue. Also, this should be introduced with a statement like, `She says, "Let's go to <destination>!"` because it's ambiguous who is speaking here.

Reply:
Used "she adds" instead of "she says", since she was already speaking just before that.